### DISCUSSION TITLE
Connect/Disconnect Keyboard HotKey

### DISCUSSION THREAD

**Discussion by karimawi** [maximmax42/Discord-CustomRP#130] [Ideas]
Sometimes I need to disable the custom rich presence to let my game or whatever other RPC apps appear as the main activity, so if possible please add a global hotkey option to connect/disconnect RPC or what would be even better if it can detect other RPC events and temporarily disconnect itself until the other RPC event ends.

**Comment by maximmax42** [comment]
Don't know if I can see other presences, but I can try adding a hotkey.

**Reply by karimawi** [reply]
Thank you very much

**Comment by marcelgustin** [comment]
When i tried it myself yesterday, it didn't work (it stayed on cusromrp), but now i opened vs code and for myself it still shows customrp everywhere, but for my friends it shows vscode...

**Reply by maximmax42** [reply]
I dunno how discord chooses which presence to show as main one, but if you open your profile, it'll show both.

**Reply by karimawi** [reply]
Yes, click on view profile, in the activity tab, you should see all of your activities when you scroll

**Comment by karimawi** [comment]
Just an idea that came to my mind while thinking about this feature request, a subtle sound feedback after using the shortcut to indicate that the disconnection/connection happened successfully would be great so, whenever you're gonna add that feature make sure to include this.

**Reply by maximmax42** [reply]
Or just a windows notification, they have a sound by default (unless you're on 7, I think).
